schematic/store/synapse.py

```diff
@@ -831,7 +831,7 @@ def format_row_annotations(self, se, sg, row, entityId, useSchemaLabel, hideBlan
         # this could create a divergence between manifest column and annotations. this should be ok for most use cases.
         # columns with special characters are outside of the schema
         metadataSyn = {}
-        blacklist_chars = ['(', ')', '.', ' ']
+        blacklist_chars = ['(', ')', '.', ' ', '-']
```

Comment:
Do we also need to make this change for [table column names](https://github.com/Sage-Bionetworks/schematic/blob/49a949137386f027daebea19c3288dbc820fc2dd/schematic/store/synapse.py#L748) as well?

Reply:
Yes - nice catch! For consistency, let's do that. I just pushed a commit. 

In the refactor we'd want to have a single list of invalid characters for Synapse annotation key names and Synapse table column names. 

(Note to self: manifest column names need to remain w/o changes, which is what the current behavior does.)

Reply:
I agree, we could put in in the utils, or create like a `store_utils` that could house other things we find specific to synapse that might change in the future